### ISSUE TITLE
q-modal-layout doesn't render properly on large screens

### ISSUE THREAD

**Issue by jean-moldovan** [quasarframework/quasar#1966]
Seems to be related to: https://github.com/quasarframework/quasar/issues/1298

```
  <q-modal v-model="opened">
    <q-modal-layout>
      <div>test</div>
    </q-modal-layout>
  </q-modal>
```

Grey background and no content is displayed on a large screens:
https://jsfiddle.net/waugrryy/1489/

**Comment by jean-moldovan** [comment]
![screen shot 2018-04-23 at 11 47 15](https://user-images.githubusercontent.com/3635352/39116066-1d8aef96-46ec-11e8-963b-35a070f7fc30.png)

**Comment by rossity** [comment]
You need to set a min-width.

**Comment by jean-moldovan** [comment]
Thanks, but I'd expect that to fallback gracefully.

**Comment by jean-moldovan** [comment]
Also. If you specify `minHeight`, this would make your modal have the fixed height. Is it possible to keep it dynamic, based on its contents?

**Comment by rstoenescu** [comment]
This should be fixed in v0.17 (to be released in a few days).
